cmd/logpaste/main.go

```diff
@@ -24,6 +24,7 @@ func main() {
 		true, "whether to display usage information on homepage")
 	perMinuteLimit := flag.Int("perminutelimit",
 		0, "number of pastes to allow per IP per minute (set to 0 to disable rate limiting)")
+	maxCharLimit := flag.Int64("maxsize", 2, "max file size as MiB")
```

Comment:
Can we reword this to "max files size in MiB"?

And can we change the var name to `maxPasteMiB`? It's a little confusing to call it `maxCharLimit` at the point that it represents the MiB limit rather than the character limit.

The conversion from MiB to characters should be the same time we rename it like:

```golang
const charactersPerMiB = 1024 * 1024
maxCharLimit := maxPasteMiB * charactersPerMiB
```

Reply:
Sounds good! For the other files, paste.go and server.go, is maxCharLimit fine in there or should I rename those too?

Reply:
Yeah, I think `main.go` is the only file that should deal in terms of max MiB. `main.go` should convert it to characters, and then the rest of the code deals in terms of max characters.

Reply:
Oh, can we also document the new flag in the README?

Reply:
Definitely, I'll be a little busy this weekend but hopefully I'll get those fixed by tomorrow evening!